static/docs/commands-reference/repro.md

```diff
@@ -80,6 +80,9 @@ specified), and updates stage files with the new checksum information.
   file(s) belong(s) to. Use `dvc pipeline show <target>.dvc` to show the parent
   pipeline of a target stage.
 
+- `-P`, `--all-pipelines` - reproduce pipeline for all the stage file(s) present
```

Comment:
@ryokugyu it should be as simple as "reproduce all pipelines, for all stage files"